pages/common/gh-issue-create.md

```diff
@@ -0,0 +1,32 @@
+# gh issue
+
+> Create GitHub issues on a repository from the command-line.
+> More information: <https://cli.github.com/manual/gh_issue_create>.
+
+- Create a new issue against the current repository interactively:
+
+`gh issue create`
+
+- Create a new issue with the `bug` label interactively:
+
+`gh issue create --label "{{bug}}"`
+
+- Create a new issue interactively and assign it to 2 users:
+
+`gh issue create --assignee {{user1,user2}}`
+
+- Create a new issue with a title, body and assigned to the current user:
+
+`gh issue create --title "{{title}}" --body "{{body}}" --assignee "{{@me}}"`
```

Comment:
Should we maybe remove `--assignee` here, since it was already introduced in the previous example? Also, is this not interactive anymore?

Reply:
If you set the required params in the CLI arguments (`title` and `body`) the command is not interacive. Also, I'm trying to say that `@me` is a wildcard for the current user of the comamnd, is not the same than `user1`.